config/globalVariable.js

```diff
@@ -32,13 +32,13 @@ module.exports = function () {
   require(ROOT_PATH + "/generics/httpStatusCodes");
 
   // Load all message constants
-  global.messageConstants = new Array
+  global.constants = new Array
   fs.readdirSync(ROOT_PATH + "/generics/message-constants")
```

Comment:
@rakeshSgr - The folder also should be **constants**

Reply:
resolved